examples/group_provisioning_demo.c

```
@@ -0,0 +1,275 @@
+               MEMBER_B, (int)recv_pkt.payload_len, recv_pkt.payload);
+
+        sodium_memzero(mkey, sizeof(mkey));
+    }
+
+    /* Outsider cannot decrypt with a different key */
+    {
+        uint8_t wrong_key[SYMMETRIC_KEY_LENGTH];
+        randombytes_buf(wrong_key, SYMMETRIC_KEY_LENGTH);
+
+        packet_t recv_pkt;
+        if (packet_deserialize(buf, len, &recv_pkt) != 0) {
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)